test/TestCases/Expressions/Add.cpp

```diff
@@ -54,4 +54,36 @@ TEMPLATE_PRODUCT_TEST_CASE ("Addition in place", "[add]", (PlatformVectorOps, VC
 
     REQUIRE_THAT (sum1, vctr::EqualsTransformedBy<addition> (srcA, srcB).withEpsilon());
     REQUIRE_THAT (sum2, vctr::EqualsTransformedBy<addition> (srcA, c).withEpsilon());
+
+    // Operator += implements special accelerated handling for the case of multiply-accumulate operations
+    auto mac1 = srcA;
+    auto mac2 = srcA;
+    //auto mac3 = srcA;
+
+    mac1 += srcB * srcC;
+    mac2 += srcB * c;
+    //mac3 += vctr::multiplyByConstant<42> << srcB;
```

Comment:
Is the `+=` operator the only way to access a multiply-accumulate operation of vctr?

Reply:
Well, you could theoretically also go for
```C++
(srcB * srcC).evalVectorOpMultiplyAccumulate (dstPointer);
```
But I don't see why you would want that 😉

Reply:
Wouldn't  it make sense also to do that in the test to _really_ make sure that, in the test,

```
 v.evalVectorOpMultiplyAccumulate (data());
```

is executed and not

```
assignExpressionTemplate (Expressions::AddVectors<extent, decltype (self), const V&> (self, v));
```
?

Reply:
It would not really make sense to me since that is an implementation detail. It's not in the scope of this test to check which variation to compute the result is used but to verify that the computation result is correct. Depending on the environment, `evalVectorOpMultiplyAccumulate` might or might not exist, e.g. for all non-apple builds without IPP this might be the case. And e.g. in case IPP is available this function is enabled for complex floats while it is only available for real floats in case only Accelerate can be used.

TLDR: I usually ensure that the dispatching works locally by setting some breakpoints before raising the PR, so I'm pretty confident that this works. The worst failure would be a slightly slower execution